tests/console/sle2docker.pm

```diff
@@ -21,8 +21,11 @@ use utils;
 use registration "install_docker_when_needed";
 use strict;
 use warnings;
+use version_utils 'is_sle';
 
 sub run {
+    return record_soft_failure('bsc#1123502 - missing Container module on SLE12-SP5') if is_sle('=12-sp5');
```

Comment:
for now?

Reply:
Yes, some modules (including contm) are not properly setup for SLE12-SP5 yet.